Resources/Prototypes/Entities/Objects/Misc/paper.yml

```
@@ -77,6 +77,12 @@
     sound: /Audio/SimpleStation14/Items/Handling/paper_drop.ogg
   - type: EmitSoundOnLand
     sound: /Audio/SimpleStation14/Items/Handling/paper_drop.ogg
+# Corvax-Printer-Start
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)